src/Generator/Parameters/QueryParameter.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Intermax\LaravelOpenApi\Generator\Parameters;
+
+use cebe\openapi\exceptions\TypeErrorException;
+use cebe\openapi\spec\Parameter;
+
+class QueryParameter implements \Intermax\LaravelOpenApi\Contracts\QueryParameter
+{
+    /**
+     * @param  string  $name
+     * @param  string  $type
+     * @param  array<mixed>|null  $options
```

Comment:
Can we be more specific than 'mixed'?
As a user of this package I'd like to know what would be correct.